parkings/tests/api/enforcement/test_valid_parking.py

```diff
@@ -75,7 +75,7 @@ def check_parking_data_keys(parking_data):
         'id', 'created_at', 'modified_at',
         'registration_number',
         'time_start', 'time_end', 'zone',
-        'operator_name',
+        'operator', 'operator_name',
```

Comment:
Could also check the contents of this field.

Reply:
Right. Does it now.